documents_context.md

```diff
@@ -0,0 +1,62 @@
+# Documents Context
+
+## Events
+
+| Type                                            | GARAIO REM         | REM | Description               |
+| ----------------------------------------------- | ------------------ | --- | ------------------------- |
+| [Documents.Document.Add](#documentsdocumentadd) | :white_check_mark: | :x: | Stores a document in GREM |
+
+### Documents.Document.Add
+
+| Field                            | Type     | Content / Remarks                                                              |
+| -------------------------------- | -------- | ------------------------------------------------------------------------------ |
+| `eventType`                      | `string` | `Documents.Document.Add`                                                       |
+| `data`                           | `hash`   |                                                                                |
+| &nbsp;&nbsp;`fetchUrl`           | `string` | an URL where the document can be downloaded from (3) **required**              |
+| &nbsp;&nbsp;`docType`            | `string` | the document type, see (1) **required**                                        |
+| &nbsp;&nbsp;`relatedToType`      | `string` | the type of the business object this document relates to, see (2) **required** |
+| &nbsp;&nbsp;`relatedToReference` | `string` | the reference of the business object this document relates to **required**     |
+| &nbsp;&nbsp;`fileName`           | `string` | the file name **required**                                                     |
+| &nbsp;&nbsp;`description`        | `string` | description of the document                                                    |
+
+Notes
+
+* (1) A generic document type is `Dokument`. To see further type names, perform the _Dokument hochladen_ action in one of the various _Dossier_ UIs in the application. There is a _Dokument-Typ_ dropdown with the type names there. This is a work in progress, [contact us](https://github.com/Garaio-REM/grem-dev-guide) if you have questions regarding document types.
+* (2) Possible values
+  * accounting
+  * property
+  * building
+  * unit
+  * invoicing-invoice (set `docType` to `KreditorRechnung` when delivering an invoice)
+  * letting-tenancy-agreement (consider setting `docType` to `MietvertragDokument` if you're delivering a signed tenancy agreement)
+* (3) If the download from `fetchUrl` fails, GREM will send back a `Documents.Document.CreateRejected` message with details.
+
+#### Example
+
+```json
+{"eventType":"Documents.Document.Add",
+  "data":{
+    "fetchUrl": "http://www.example.com/document",
+    "docType": "Dokument",
+    "relatedToType": "accounting",
+    "relatedToReference": "123805",
+    "fileName": "document.pdf"
+  }
+}
```

Comment:
Can you clarify what kind of Document this would create? 

I think I need more examples and the resulting documents to understand this API and what it would create. 

Practical documents I can think of are:
* signed Mietvertraege
* AVK Dokuments
* uploaded Belege to be associated with a Buchung
* uploaded Dokuments to be associated with a Dossier

I would be most comfortable if we run this API past Pesche and Steph - they are good at thinking of Documents and what are done with them from the business perspective (and to some extent the technical too).